point_cloud_transport/include/point_cloud_transport/subscriber_plugin.hpp

```diff
@@ -251,6 +312,16 @@ class POINT_CLOUD_TRANSPORT_PUBLIC SubscriberPlugin
     const Callback & callback,
     rmw_qos_profile_t custom_qos = rmw_qos_profile_default) = 0;
```

Comment:
Provide implementation?
```suggestion
    rmw_qos_profile_t custom_qos = rmw_qos_profile_default)
  {
    subscribeImpl(
        node_interfaces,
        base_topic, callback,
        rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(custom_qos), custom_qos));
  }
```

Reply:
Missed this one :')